joomla/joomla-cms
# Send Test Mail button - Ajax call has always smtpauth = 1

Pull Request for Issue #9768 / #9803
#### Summary of Changes

Send Test Mail button does not correctly evaluate "SMTP Authentication" radio. Then, mails are sent via SMTP even when "SMTP Authentication" is disabled.

Fix: smtpauth : $('input[name="jform[smtpauth]"]:checked').val(),

The problem is found when your fill all SMTP credentials and keep "SMTP Authentication"=No... it applies the credentials even when it is disabled.

In this PR, I have modified the code to really check for the "SMTP Authentication" radio value.

_Steps to reproduce the issue_
- Fill all SMTP fields in Mail Settings for a SMTP server that requires authentication
- But, don't enable SMTP Authentication
- Send Test Mail button sends emails even when the radio is disabled
#### Testing Instructions
1. Fill all SMTP fields in Mail Settings for a SMTP server that requires authentication
2. Test it, click on 'Send Test Mail' button to receive an email
3. Disable "SMTP Authentication" radio
4. Test it, click on 'Send Test Mail' button ... email must not be received and error is shown about the failed authentication
